MainProgram/AlgorithmsTests/SieveOfEratosthenesTest.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections.Generic;

```

Comment:
All unit tests should be added to the `UnitTest` project. The `MainProgram` project is deprecated and will be removed soon.